### DISCUSSION TITLE
Interpreting data: can velocity estimate catch self-renewal?

### DISCUSSION THREAD

**Discussion by france-hub** [theislab/scvelo#870] [Q&A]
Hello scvelo team!
Thank you for the wonderful package. I have a question on how to interpret my data (it's not actually an issue with your package). 
Short premise: I am an MD and I apologize in advance for the naive question. 
I am studying single cell data on T cells and I am particularly interested in their differentiation. Velocity estimate fundamentally reproduce the directionality of the trajectory inference. However, there are some bidirectional arrows in one subset (celltype number 2, green subset in the plot). These cells, looking at their phenotype, are stem like cells and by definition they should self-renew. Can the bi-directional arrows be explained with this feature of renewing themselves? 
Thanks!
Francesco 

![image](https://user-images.githubusercontent.com/56587709/165832976-fce67352-2b20-4a80-afc3-5033f76dfc06.png)

**Comment by WeilerP** [comment]
@france-hub, projected arrows in a latent space representation should not be over interpreted. To check if your data is suitable for RNA velocity analysis, you should check if phase portraits exhibit the expected/required shape. If this is the case, you can use tools such as CellRank to study cellular lineages and fate.
